opencti-platform/opencti-front/src/private/components/import/ImportContent.jsx

```diff
@@ -199,7 +201,7 @@ class ImportContentComponent extends Component {
   }
 
   handleCloseImport() {
-    this.setState({ fileToImport: null });
+    this.setState({ ...initState });
```

Comment:
No need for spreding

Reply:
Ok will remove it. Thank you.

Reply:
I changed it as I explained above. But I try without the spreading, and it did not rerender and we are blocked we the modal open. I think using the object with spread, create the same object properties with a different reference and make the component rerender.

Reply:
yes, spreading will just shallow copy the properties in a new object, so the ref does change and a rerender is triggered